layouts/partials/navigation.html

```diff
@@ -5,26 +5,18 @@
       {{ $logoAlt:= site.Params.logo_alt }}
       {{ $orgName:= site.Params.brand.parent_org_name }}
       {{ $orgUrl:= site.Params.brand.parent_org_url }}
-      <a class="navbar-brand px-2"
-        href="{{ if (or $logo $logoAlt) }}{{ $orgUrl }}{{else}}{{ site.BaseURL | relLangURL }}{{ end }}">
+
+      <div class="text-center">
+        <a class="navbar-brand" href="{{ if (or $logo $logoAlt) }}{{ $orgUrl }}{{ else }}{{ site.BaseURL | relLangURL }}{{ end }}">
         {{ if (or $logo $logoAlt) }}
-        {{ if .IsHome }}
```

Comment:
Removing this line makes the primary logo and alternate logo redundant. Setting an alternate logo will always override a normal logo. Allowing a webmaster to set both seems unnecessary.

Maybe this is a good time to consider whether we need to support an alternate logo in the first place. Is it easier to drop the `logoAlt` and simplify the code we need to maintain?

Reply:
Yes, I also thought the same. We should remove the `logoAlt`

Reply:
Done in commit 1cc8bfe39625cc5559c96fa1e4e449c4ebb19d01.